### ISSUE TITLE
2025-01-30, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#67]
# Today's Arxiv Papers

(1) [Dialogue is Better Than Monologue: Instructing Medical LLMs via Strategical Conversations](http://arxiv.org/abs/2501.17860v1), 2025-01-29

(2) [Improving Your Model Ranking on Chatbot Arena by Vote Rigging](http://arxiv.org/abs/2501.17858v1), 2025-01-29

(3) [Learning Beyond the Surface: How Far Can Continual Pre-Training with LoRA Enhance LLMs' Domain-Specific Insight Learning?](http://arxiv.org/abs/2501.17840v1), 2025-01-29

(4) [LEKA:LLM-Enhanced Knowledge Augmentation](http://arxiv.org/abs/2501.17802v1), 2025-01-29

(5) [Leveraging Multimodal LLM for Inspirational User Interface Search](http://dx.doi.org/10.1145/3706598.3714213), 2025-01-29

(6) [BreezyVoice: Adapting TTS for Taiwanese Mandarin with Enhanced Polyphone Disambiguation -- Challenges and Insights](http://arxiv.org/abs/2501.17790v1), 2025-01-29

(7) [Reasoning Over the Glyphs: Evaluation of LLM's Decipherment of Rare Scripts](http://arxiv.org/abs/2501.17785v1), 2025-01-29

(8) [AdditiveLLM: Large Language Models Predict Defects in Additive Manufacturing](http://arxiv.org/abs/2501.17784v1), 2025-01-29

(9) [2SSP: A Two-Stage Framework for Structured Pruning of LLMs](http://arxiv.org/abs/2501.17771v1), 2025-01-29

(10) [Hybrid Graphs for Table-and-Text based Question Answering using LLMs](http://arxiv.org/abs/2501.17767v1), 2025-01-29

(11) [On the Partitioning of GPU Power among Multi-Instances](http://arxiv.org/abs/2501.17752v1), 2025-01-29

(12) [Early External Safety Testing of OpenAI's o3-mini: Insights from the Pre-Deployment Evaluation](http://arxiv.org/abs/2501.17749v1), 2025-01-29

(13) [Using Code Generation to Solve Open Instances of Combinatorial Design Problems](http://arxiv.org/abs/2501.17725v1), 2025-01-29

(14) [RICoTA: Red-teaming of In-the-wild Conversation with Test Attempts](http://arxiv.org/abs/2501.17715v1), 2025-01-29

(15) [Planning with Vision-Language Models and a Use Case in Robot-Assisted Teaching](http://arxiv.org/abs/2501.17665v1), 2025-01-29

(16) [In-Context Meta LoRA Generation](http://arxiv.org/abs/2501.17635v1), 2025-01-29

(17) [The Imitation Game According To Turing](http://arxiv.org/abs/2501.17629v1), 2025-01-29

(18) [Uncertainty Quantification and Decomposition for LLM-based Recommendation](http://dx.doi.org/10.1145/3696410.3714601), 2025-01-29

(19) [Structured Context Recomposition for Large Language Models Using Probabilistic Layer Realignment](http://arxiv.org/abs/2501.17617v1), 2025-01-29

(20) [Semantic Consistency Regularization with Large Language Models for Semi-supervised Sentiment Analysis](http://arxiv.org/abs/2501.17598v1), 2025-01-29

(21) [GLLM: Self-Corrective G-Code Generation using Large Language Models with User Feedback](http://arxiv.org/abs/2501.17584v1), 2025-01-29

(22) [CSEval: Towards Automated, Multi-Dimensional, and Reference-Free Counterspeech Evaluation using Auto-Calibrated LLMs](http://arxiv.org/abs/2501.17581v1), 2025-01-29

(23) [Query-Aware Learnable Graph Pooling Tokens as Prompt for Large Language Models](http://arxiv.org/abs/2501.17549v1), 2025-01-29

(24) [Is Conversational XAI All You Need? Human-AI Decision Making With a Conversational XAI Assistant](http://dx.doi.org/10.1145/3708359.3712133), 2025-01-29

(25) [Towards Supporting Penetration Testing Education with Large Language Models: an Evaluation and Comparison](http://arxiv.org/abs/2501.17539v1), 2025-01-29

(26) [LLM Assistance for Pediatric Depression](http://arxiv.org/abs/2501.17510v1), 2025-01-29

(27) [Neural Spelling: A Spell-Based BCI System for Language Neural Decoding](http://arxiv.org/abs/2501.17489v1), 2025-01-29

(28) [DFPE: A Diverse Fingerprint Ensemble for Enhancing LLM Performance](http://arxiv.org/abs/2501.17479v1), 2025-01-29

(29) [AugmenTest: Enhancing Tests with LLM-Driven Oracles](http://arxiv.org/abs/2501.17461v1), 2025-01-29

(30) [Large Language Models for Single-Step and Multi-Step Flight Trajectory Prediction](http://arxiv.org/abs/2501.17459v1), 2025-01-29

(31) [Virus: Harmful Fine-tuning Attack for Large Language Models Bypassing Guardrail Moderation](http://arxiv.org/abs/2501.17433v1), 2025-01-29

(32) [Actions Speak Louder than Words: Agent Decisions Reveal Implicit Biases in Language Models](http://arxiv.org/abs/2501.17420v1), 2025-01-29

(33) [General Scene Adaptation for Vision-and-Language Navigation](http://arxiv.org/abs/2501.17403v1), 2025-01-29

(34) [MultiChallenge: A Realistic Multi-Turn Conversation Evaluation Benchmark Challenging to Frontier LLMs](http://arxiv.org/abs/2501.17399v1), 2025-01-29

(35) [Learning Free Token Reduction for Multi-Modal LLM](http://arxiv.org/abs/2501.17391v1), 2025-01-29

(36) [Context-Aware Semantic Recomposition Mechanism for Large Language Models](http://arxiv.org/abs/2501.17386v1), 2025-01-29

(37) [Attribution analysis of legal language as used by LLM](http://arxiv.org/abs/2501.17330v1), 2025-01-28

(38) [Memorize and Rank: Elevating Large Language Models for Clinical Diagnosis Prediction](http://arxiv.org/abs/2501.17326v1), 2025-01-28

(39) [A sketch of an AI control safety case](http://arxiv.org/abs/2501.17315v1), 2025-01-28

(40) [Probing LLM World Models: Enhancing Guesstimation with Wisdom of Crowds Decoding](http://arxiv.org/abs/2501.17310v1), 2025-01-28

(41) ["Ownership, Not Just Happy Talk": Co-Designing a Participatory Large Language Model for Journalism](http://arxiv.org/abs/2501.17299v1), 2025-01-28

(42) [Mitigating Hallucinated Translations in Large Language Models with Hallucination-focused Preference Optimization](http://arxiv.org/abs/2501.17295v1), 2025-01-28

(43) [Fine-Tuning Open-Source Large Language Models to Improve Their Performance on Radiation Oncology Tasks: A Feasibility Study to Investigate Their Potential Clinical Applications in Radiation Oncology](http://arxiv.org/abs/2501.17286v1), 2025-01-28

(44) [From Natural Language to Extensive-Form Game Representations](http://arxiv.org/abs/2501.17282v1), 2025-01-28

(45) [Engineering Point Defects in MoS2 for Tailored Material Properties using Large Language Models](http://arxiv.org/abs/2501.17279v1), 2025-01-28

(46) [Tailored Truths: Optimizing LLM Persuasion with Personalization and Fabricated Statistics](http://arxiv.org/abs/2501.17273v1), 2025-01-28

(47) [AxBench: Steering LLMs? Even Simple Baselines Outperform Sparse Autoencoders](http://arxiv.org/abs/2501.17148v2), 2025-01-28

(48) [FactCG: Enhancing Fact Checkers with Graph-Based Multi-Hop Data](http://arxiv.org/abs/2501.17144v1), 2025-01-28

(49) [ASTRAL: Automated Safety Testing of Large Language Models](http://arxiv.org/abs/2501.17132v1), 2025-01-28

(50) [Histoires Morales: A French Dataset for Assessing Moral Alignment](http://arxiv.org/abs/2501.17117v1), 2025-01-28

(51) [Optimizing Large Language Model Training Using FP4 Quantization](http://arxiv.org/abs/2501.17116v1), 2025-01-28

(52) [Unlocking Transparent Alignment Through Enhanced Inverse Constitutional AI for Principle Extraction](http://arxiv.org/abs/2501.17112v1), 2025-01-28

(53) [Token-by-Token Regeneration and Domain Biases: A Benchmark of LLMs on Advanced Mathematical Problem-Solving](http://arxiv.org/abs/2501.17084v1), 2025-01-28

(54) [Enhanced Retrieval of Long Documents: Leveraging Fine-Grained Block Representations with Large Language Models](http://arxiv.org/abs/2501.17039v1), 2025-01-28

(55) [Challenges in Ensuring AI Safety in DeepSeek-R1 Models: The Shortcomings of Reinforcement Learning Strategies](http://arxiv.org/abs/2501.17030v1), 2025-01-28

(56) [Automated Refactoring of Non-Idiomatic Python Code: A Differentiated Replication with LLMs](http://arxiv.org/abs/2501.17024v1), 2025-01-28

(57) [Mobile Manipulation Instruction Generation from Multiple Images with Automatic Metric Enhancement](http://arxiv.org/abs/2501.17022v1), 2025-01-28

(58) [Large Language Models for Code Generation: The Practitioners Perspective](http://arxiv.org/abs/2501.16998v1), 2025-01-28

(59) [Artificial Intelligence Clones](http://arxiv.org/abs/2501.16996v1), 2025-01-28

(60) [Over-Tokenized Transformer: Vocabulary is Generally Worth Scaling](http://arxiv.org/abs/2501.16975v1), 2025-01-28

(61) [Instantiation-based Formalization of Logical Reasoning Tasks using Language Models and Logical Solvers](http://arxiv.org/abs/2501.16961v1), 2025-01-28

(62) [Multiple Abstraction Level Retrieve Augment Generation](http://arxiv.org/abs/2501.16952v1), 2025-01-28

(63) [ToolFactory: Automating Tool Generation by Leveraging LLM to Understand REST API Documentations](http://arxiv.org/abs/2501.16945v1), 2025-01-28

(64) [Detecting harassment and defamation in cyberbullying with emotion-adaptive training](http://arxiv.org/abs/2501.16925v1), 2025-01-28

(65) [RDMM: Fine-Tuned LLM Models for On-Device Robotic Decision Making with Enhanced Contextual Awareness in Specific Domains](http://arxiv.org/abs/2501.16899v1), 2025-01-28

(66) [Irony Detection, Reasoning and Understanding in Zero-shot Learning](http://arxiv.org/abs/2501.16884v1), 2025-01-28

(67) [JRE-L: Journalist, Reader, and Editor LLMs in the Loop for Science Journalism for the General Audience](http://arxiv.org/abs/2501.16865v1), 2025-01-28

(68) [Comparing Human and LLM Generated Code: The Jury is Still Out!](http://arxiv.org/abs/2501.16857v1), 2025-01-28

(69) [Adapting Network Information to Semantics for Generalizable and Plug-and-Play Multi-Scenario Network Diagnosis](http://arxiv.org/abs/2501.16842v1), 2025-01-28

(70) [Misspellings in Natural Language Processing: A survey](http://arxiv.org/abs/2501.16836v1), 2025-01-28

(71) [Exponential Family Attention](http://arxiv.org/abs/2501.16790v1), 2025-01-28

(72) [Exploring the Role of Explicit Temporal Modeling in Multimodal Large Language Models for Video Understanding](http://arxiv.org/abs/2501.16786v1), 2025-01-28

(73) [TORCHLIGHT: Shedding LIGHT on Real-World Attacks on Cloudless IoT Devices Concealed within the Tor Network](http://arxiv.org/abs/2501.16784v1), 2025-01-28

(74) [A Stochastic Dynamical Theory of LLM Self-Adversariality: Modeling Severity Drift as a Critical Process](http://arxiv.org/abs/2501.16783v1), 2025-01-28

(75) [HateBench: Benchmarking Hate Speech Detectors on LLM-Generated Content and Hate Campaigns](http://arxiv.org/abs/2501.16750v1), 2025-01-28

(76) [Through the Prism of Culture: Evaluating LLMs' Understanding of Indian Subcultures and Traditions](http://arxiv.org/abs/2501.16748v1), 2025-01-28

(77) [LLM Assisted Anomaly Detection Service for Site Reliability Engineers: Enhancing Cloud Infrastructure Resilience](http://arxiv.org/abs/2501.16744v1), 2025-01-28

(78) [Integrating Reinforcement Learning and AI Agents for Adaptive Robotic Interaction and Assistance in Dementia Care](http://arxiv.org/abs/2501.17206v1), 2025-01-28

(79) [Distilling Large Language Models for Network Active Queue Management](http://arxiv.org/abs/2501.16734v1), 2025-01-28

(80) [xJailbreak: Representation Space Guided Reinforcement Learning for Interpretable LLM Jailbreaking](http://arxiv.org/abs/2501.16727v1), 2025-01-28

(81) [3D-MoE: A Mixture-of-Experts Multi-modal LLM for 3D Vision and Pose Diffusion via Rectified Flow](http://arxiv.org/abs/2501.16698v1), 2025-01-28

(82) [Optimizing Code Runtime Performance through Context-Aware Retrieval-Augmented Generation](http://arxiv.org/abs/2501.16692v2), 2025-01-28

(83) [MACI: Multi-Agent Collaborative Intelligence for Adaptive Reasoning and Temporal Planning](http://arxiv.org/abs/2501.16689v2), 2025-01-28

(84) [MME-Industry: A Cross-Industry Multimodal Evaluation Benchmark](http://arxiv.org/abs/2501.16688v1), 2025-01-28

(85) [Auto-Differentiating Any LLM Workflow: A Farewell to Manual Prompting](http://arxiv.org/abs/2501.16673v1), 2025-01-28

(86) [VeriFact: Verifying Facts in LLM-Generated Clinical Text with Electronic Health Records](http://arxiv.org/abs/2501.16672v1), 2025-01-28

(87) [Jupybara: Operationalizing a Design Space for Actionable Data Analysis and Storytelling with LLMs](http://arxiv.org/abs/2501.16661v1), 2025-01-28

(88) [Contextual Reinforcement in Multimodal Token Compression for Large Language Models](http://arxiv.org/abs/2501.16658v1), 2025-01-28

(89) [Large Language Model Critics for Execution-Free Evaluation of Code Changes](http://arxiv.org/abs/2501.16655v1), 2025-01-28

(90) [DOCS: Quantifying Weight Similarity for Deeper Insights into Large Language Models](http://arxiv.org/abs/2501.16650v1), 2025-01-28

(91) [An LLM Benchmark for Addressee Recognition in Multi-modal Multi-party Dialogue](http://arxiv.org/abs/2501.16643v1), 2025-01-28

(92) [Why Do We Laugh? Annotation and Taxonomy Generation for Laughable Contexts in Spontaneous Text Conversation](http://arxiv.org/abs/2501.16635v1), 2025-01-28

(93) [CHiP: Cross-modal Hierarchical Direct Preference Optimization for Multimodal LLMs](http://arxiv.org/abs/2501.16629v1), 2025-01-28

(94) [Sparse Autoencoders Trained on the Same Data Learn Different Features](http://arxiv.org/abs/2501.16615v1), 2025-01-28

(95) [MCTS-SQL: An Effective Framework for Text-to-SQL with Monte Carlo Tree Search](http://arxiv.org/abs/2501.16607v1), 2025-01-28

(96) [Fine-Tuned Language Models as Space Systems Controllers](http://arxiv.org/abs/2501.16588v1), 2025-01-28
